tools/find_forbidden_words_in_repo.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
 
-words=$"colour\|visualisation"
+words=$"colour\|visualisation\|openai.api_base"
 words_finder=$(grep -Irwno --exclude=find_forbidden_words_in_repo.sh --exclude-dir={.git,*cache*,*node_modules*,venv} . -e "$words")
 
 if [[ $words_finder ]]
 then
-  echo "Incorrect spelling for $words:
+  echo "Incorrect spelling or forbidden word for $words:
```

Comment:
probably we can do two check separately with different message, something as below? 

btw, is color and visualization the correct so we are forbidden those two?
```
#!/bin/bash

# General words check
general_words="colour\|visualisation"
general_words_finder=$(grep -Irwno --exclude=find_forbidden_words_in_repo.sh --exclude-dir={.git,*cache*,*node_modules*,venv} . -e "$general_words")

# openai.api_base check
api_base_word="openai.api_base"
api_base_finder=$(grep -Irwno --exclude=find_forbidden_words_in_repo.sh --exclude-dir={.git,*cache*,*node_modules*,venv} . -e "$api_base_word")

# If general forbidden words are found
if [[ $general_words_finder ]]; then
  echo "Incorrect spelling or forbidden word for $general_words:
$general_words_finder"
  exit 1
fi

# If openai.api_base is found
if [[ $api_base_finder ]]; then
  echo "Please remove URL base for development:
$api_base_finder"
  exit 1
fi
```

Reply:
colour and visualisation are British spelling and we opted for American spelling - preferably I would love to have a linter that just checks for British/American spelling instead of adding these words here manually. But I think there wasn't any available - maybe quickly double-check that?